index.html

```diff
@@ -674,15 +674,20 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 the last request.</p>
 </aside>
 <p data-dfn-for="PerformanceResourceTiming">On getting, the
-<dfn>responseEnd</dfn> attribute MUST return as follows:</p>
+<dfn>responseEnd</dfn> attribute MUST return the result of running
```

Comment:
nit: of running the algorithm to <a>get ...

Reply:
sure